theano/d3viz/d3viz.py

```diff
@@ -78,7 +79,10 @@ def d3viz(fct, outfile, copy_deps=True, *args, **kwargs):
     # Create DOT graph
     formatter = PyDotFormatter(*args, **kwargs)
     graph = formatter(fct)
-    dot_graph = escape_quotes(str(graph.create_dot())).replace('\n', '').replace('\r', '')
+    dot_graph_raw = graph.create_dot()
+    if not six.PY2:
+        dot_graph_raw = dot_graph_raw.decode('utf8')
+    dot_graph = escape_quotes(dot_graph_raw).replace('\n', '').replace('\r', '')
```

Comment:
Travis skiped the associated tests as pydot wasn't installed. Do you want to modify the file .travis.yml
Change this line:
  - if [[ $TRAVIS_PYTHON_VERSION == '2.6' ]]; then pip install pydot; fi

to:
  - pip install pydot-ng

I think it also exist for python 2.